ultra/ultra/utils/common.py

```diff
@@ -145,7 +144,7 @@ def copy_source_code(srcs, dst):
 
 
 def resize_im(im, size):
-    im = (resize(im, size, anti_aliasing=True, mode="constant") * 255.0).astype(
+    im = (cv2.resize(im, size) * 255.0).astype(
```

Comment:
How important is anti-aliasing? Should I add a blur to mimic this here?

Reply:
`cv2.resize` has anti-aliasing integrated. `resize(..,interpolation=..)` works with the following:
- 0 INTER_NEAREST
- 1 INTER_LINEAR
- 2 INTER_AREA
- 3 INTER_CUBIC
- 4 INTER_LANCZOS4

Reply:
I am honestly unsure what the implications are.

Reply:
I am honestly unsure what the implications are.